lib/fill/index.js

```diff
@@ -0,0 +1,9 @@
+import curryN from '../curryN'
+
+export default curryN(2, function fill(arr, value) {
+  return Array.isArray(arr)
+    ? value
+      ? arr.fill(value)
```

Comment:
It mutates arr

Reply:
Original lodash method mutates array, but I think not mutating is better

![lodash_documentation](https://user-images.githubusercontent.com/25172855/35428460-3e8eb782-0280-11e8-8f52-46acc252891f.png)